s4sim/hardware/config.py

```diff
@@ -682,6 +682,19 @@ def get_example():
         "SPLAT_HF": ["SPLAT_f220", "SPLAT_f280"],
         "SAT_HF": ["SAT_f220", "SAT_f280"],
     }
+    pins = {
```

Comment:
What are these? Perhaps adding a comment line would clarify it.

Reply:
Added a comment